support/client/lib/vwf/model/jPlayer.js

```diff
@@ -361,12 +362,21 @@ define( [
 
                 switch( methodName ) {
                     
-                    case "play":
+                    case "load":
+                        if( node.url ) {
+                            node.jPlayerElement.jPlayer( "load" ); 
+                            this.logger.infox( "Loading!" ); 
+                        } else {
+                            this.logger.errorx( "No URL given!" ); 
+                        }
+                        break;
 
-                        if( methodParameters ){
-                            this.kernel.setProperty( node.ID, "url", methodParameters );
+                    case "play":
+                        if( node.url ) {
+                            node.jPlayerElement.jPlayer( "play" ); 
+                        } else {
+                            this.logger.errorx( "No URL given!" ); 
```

Comment:
Should play require a URL?  Doesn't it just play the video that was previously set using the load command?